app/schema/portal/search/basic.js

```diff
@@ -0,0 +1,12 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  searchTerms: Joi.string().trim().required().max(100).messages({
+    'string.empty': 'Enter some search terms'
```

Comment:
Not sure on this error message. Was this supplied by content designer? If not we can fix later once we have content design for it.

Reply:
Yeah, don't think anyone supplied it (but me) but it seems to match with the hint of 'Enter one or more search terms separated by spaces'